src/azure-cli/HISTORY.rst

```diff
@@ -25,6 +25,10 @@ Release History
 
 * Support app creation/update with the new sku name ST0, ST1, ST2.
 
+**Key Vault**
+
+* Add a new command `az keyvault key download` for downloading keys.
```

Comment:
in format of .... and I think `History.rst` file is auto-generated now, right? pls confirm with @fengzhou-msft

Reply:
@yungezz Just confirmed with him offline, we still need to modify `HISTORY.rst` manually in this sprint (S164). After the early Feb release, this file will be generated automatically.